src/main/kotlin/app/revanced/patches/reddit/misc/disablescreenshotpopup/fingerprints/DisableScreenshotPopupFingerprint.kt

```diff
@@ -0,0 +1,21 @@
+package app.revanced.patches.reddit.misc.disablescreenshotpopup.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.Opcode
+
+object DisableScreenshotPopupFingerprint : MethodFingerprint(
+    "V",
+    opcodes = listOf(
```

Comment:
This pattern may break in the future, shat about adding the parameters or strings?

Reply:
it has no strings. this is the parameter (Landroidx/compose/runtime/e;I). what should be parameters be for patch since it only looks at first character?

Reply:
You can use a list of "L" and "I". But recently this changed you can also use the full type string.